jina/executors/indexers/keyvalue.py

```diff
@@ -96,23 +152,10 @@ def add(self, keys: Iterable[str], values: Iterable[bytes], *args, **kwargs) ->
         :param args: extra arguments
         :param kwargs: keyword arguments
         """
-        if not keys:
+        if not len(keys):
```

Comment:
`keys` is an iterable object and `len()` doesn't always work. 

Reference: https://docs.python.org/3/library/functions.html#any
```suggestion
        if not any(keys):
```

Reply:
Ah yes, thanks for the reference